Tutorial 3 - Filtrar e Combinar VCFs

VicAzv opened the pull request:
Tutorial 3 onde foi adicionado uma task com Hard Filter para SNP, uma task com Hard Filter para INDEL e uma task para combinar os VCFs gerados dessas duas tasks anteriores.

IsadoraRFMuniz committed:
Tutorial 3 - Filtrar e Combinar VCFs

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+            refDict = refDict,
+            sampleName = sampleName,
+            type = "SNP", 
+                rawVcf = testando
```
um recuo a mais.

VicAzv replied:
Não entendi, eu to contando a partir do "t" do "type"  e da 4 espaços certinho

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+            refDict = refDict,
+            sampleName = sampleName,
+            type = "INDEL", 
+                rawVcf = testando 
```
recuo a mais

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+            refDict = refDict, 
+            rawSNPs = selectSNPs.rawSubset
+    }
+    call HardFilterIndel {
```
falta uma linha vazia

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+            refDict = refDict,
+            rawIndels = selectIndels.rawSubset
+    }
+    call Combine {
```
falta uma linha vazia

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+        File filteredSNPs
+        File filteredIndels
+    }
+    command <<<
```
adicionar uma linha vazia

desouzareis commented on simple-variant-discovery.wdl:
```diff
@@ -0,0 +1,199 @@
+
+    output {
+        File rawVcf = "~{sampleName}.raw.indels.snps.vcf"
+    }
```
Pode ser legal adicionar a sessão parameter_meta nas tasks para descrever os inputs que elas recebem e outputs que elas geram.

desouzareis replied:
A variável do output não é "output" e sim "rawVcf", mesma coisa para os parameter_meta abaixo. ex: rawSubset, filteredSNPs, filteredIndels, filteredVCF

desouzareis reviewed (changes requested):
Parabéns, muito bom! Apenas alguns pequenos detalhes

VicAzv committed:
Correções

Feito algumas mudanças exigidas e adição do parameter_meta para cada uma das tasks

VicAzv committed:
Mudanças no parameter_meta

VicAzv committed:
Mudanças no parameter_meta

Mudança do "output" em todos os parameter_meta

desouzareis reviewed (approved):
Top!
